examples/ActionGroup/ActionGroup.Tests.ps1

```diff
@@ -2,31 +2,69 @@ BeforeAll {
   Import-Module Az.InfrastructureTesting
 }
 
+$resourceType = "ActionGroup"
+$resourceName = "sampleaction"
+$rgName = "rg-test"
+
+Describe 'Verify Action Group with Confirm-AzBPResource' {
+  it 'Should contain an actiongroup named sampleaction' {
+    #arrange
+    $params = @{
+      ResourceType      = $resourceType
+      ResourceName      = $resourceName
+      ResourceGroupName = $rgName
+    }
+
+    #act
+    $result = Confirm-AzBPResource @params
+
+    #assert
+    $result.Success | Should -Be $true
+  }
+
+  it 'Should contain an actiongroup named sampleaction' {
+    #arrange
+    $params = @{
+      ResourceType      = $resourceType
+      ResourceName      = $resourceName
+      ResourceGroupName = $rgName
+      PropertyKey       = "GroupShortName"
+      PropertyValue     = $resourceName
+    }
+
+    #act
+    $result = Confirm-AzBPResource @params
+
+    #assert
+    $result.Success | Should -Be $true
+  }
+}
+
 Describe 'Verify Action Group Exists' {
-  it 'Should contain a action group named sampleaction' {
+  it 'Should contain an action group named sampleaction' {
     #arrange
-    $resourceGroupName = "rg-test"
-    $actionGroupName = "sampleaction"
+    $rgName = $rgName
+    $actionGroupName = $resourceName
 
     #act
-    $result = Confirm-AzBPActionGroup -ResourceGroupName $resourceGroupName -ActionGroupName $actionGroupName
+    $result = Confirm-AzBPActionGroup -ResourceGroupName $rgName -ActionGroupName $actionGroupName
 
     #assert
     $result.Success | Should -Be $true
```

Comment:
Curious if there is a reason we do both the result Success and result should be deployed as 2 separate tests. Would it make sense to combine into a single test? With both assert statements?   

I also wonder if we could combine some of the other tests, like the resource group and location ones?  Ok with you not addressing it in this PR and having tabling it for a design session

Reply:
As a best practice there should be `one test`:`one assert`. This isn't always practical or pretty, but it is a goal. In this case (as in the previous comment) the purpose of each test independently is different, but the results are the same.